operator_manifest/operator.py

```diff
@@ -376,6 +376,10 @@ def set_related_images(self):
         """
         named_pullspecs = self._named_pullspecs()
 
+        if not named_pullspecs:
+            log.info("No pullspecs, skipping updates of relatedImages section")
```

Comment:
>  Don't set relatedImages if it already exists 

This looks conflicts with this logic and the info message.

Reply:
@tkdchen I believe all the changes in here were already applied to the atomic-reactor module, this is literally just a rebase.

> Instead of doing such rebase, how about abstract the code and make a shared library for the operator manifests?

That's the plan, we just have not got around to doing that yet :D

Reply:
Right. I'm not changing any of the code on purpose. It's already painful as is to port these things back :)